### ISSUE TITLE
Logo program for popular R packages

### ISSUE THREAD

**Issue by gaborcsardi** [RConsortium/wishlist#7]
To help maintainers market their R packages, it would be nice if the more popular R packages had logos. 

Many of them do, but many don't. We would target maintainers that otherwise would not pay for a logo, or they don't want to deal designers, etc. E.g. jsonlite and curl would be nice candidates, and Rcpp is probably the most obvious one. 

I am not sure how one would work out what is popular, and who would get a logo, but maybe there is a good way.

Obviously, if the maintainer does not want a logo, then we would not force them....

**Comment by jeroen** [comment]
For opencpu packages (which are also R packages) the package author can use `/inst/icon.png` to customize the icon displayed in the graphical repository index: https://www.opencpu.org/apps.html

**Comment by gaborcsardi** [comment]
Sorry, maybe it is not clear what I wanted to suggest here. :)

So my suggestion is that we have many (~10-20) small grants that package maintainers can apply for, and then the logo program would pay a designer to create a logo for their package. 

@jeroenooms But having a standard place for a logo is a great idea indeed, so CRAN, MRAN, http://www.r-pkg.org/, etc. would be able to show it on the package page, or on these package lists: http://www.r-pkg.org/downloaded

**Comment by hilaryparker** [comment]
oh, I like this a lot. Will ponder creating a proposal.

**Comment by karthik** [comment]
@gaborcsardi 
I like the general idea. Much of the Tidyverse ™ ecosystem and friends already have logos.

> I am not sure how one would work out what is popular, and who would get a logo, but maybe there is a good way.

If packages are chosen by popularity (downloads, reverse dependencies) aren’t those packages already very popular which means they don't need as much marketing (e.g. Rcpp)?

Would it then help to build some sort of javascript logo generator that any package maintainer could use to build one for the project? The impact would be larger and help projects that need promotion. Tying into a noun project license could also help.

**Comment by karthik** [comment]
@gaborcsardi One other idea is for projects to apply for the logo program (once some funding has been allocated). Projects that are selected (on a diverse range of criteria) could then work with a designer (or service, see below) that the consortium has already paid for.

Other rough notes:
* 99 Designs produces fairly decent logos that would be nice for most packages. We even used it for the useR conference last year. The grant could cover the cost of [99 Designs](https://99designs.com/) ($299) for each selected project which would make budgeting and designing fairly straightforward.

**Comment by gaborcsardi** [comment]
@karthik Yeah, sg. like that. I guess if we buy ~10-20 logos from the same designer, we can also get some discount.
